sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyClient.cs

```diff
@@ -1334,5 +1335,32 @@ public virtual async Task<Response<ReleaseKeyResult>> ReleaseKeyAsync(string nam
                 throw;
             }
         }
+
+        /// <summary>
+        /// Get a <see cref="CryptographyClient"/> for the given key.
+        /// </summary>
+        /// <param name="name">The name of the key used to perform cryptographic operations.</param>
+        /// <param name="version">Optional version of the key used to perform cryptographic operations.</param>
+        /// <returns>A <see cref="CryptographyClient"/> using the same options and pipeline as this <see cref="KeyClient"/>.</returns>
+        /// <exception cref="ArgumentException"><paramref name="name"/> is an empty string.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="name"/> is null.</exception>
+        public virtual CryptographyClient GetCryptographyClient(string name, string version = null)
```

Comment:
I'm also interested in the specification of version here.  Isn't version specified in ClientOptions?  Does this override that?  Would you want a CryptoClient that targets a different version than the main KVClient?  (Or am I misunderstanding how KV works?)

Reply:
It's the key-version. The other version is the service-version that is used to create the key-client

Reply:
It's not that version. As the `<param>` in line 1343 notes, it's the version of the key you want to use. It's the same for all our methods.